utils/translation_service.py

```
@@ -0,0 +1,245 @@
+            # 调用LLM进行翻译
+            response = await llm.ainvoke([
+                SystemMessage(content=system_prompt),
+                HumanMessage(content=user_prompt)
+            ])
+            
+            translated = response.content.strip()
+            
+            # 保存到缓存
+            self._save_to_cache(text, target_lang, translated)
+            
+            logger.debug(f"翻译服务：'{text[:50]}...' -> '{translated[:50]}...' ({target_lang})")
```

Ruff (0.14.8)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)